bio/ngsbits/sampleancestry/wrapper.py

```
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+"""Snakemake wrapper for NGS-bits SampleAncestry"""
+
+__author__ = "Thibault Dayris"
+__copyright__ = "Copyright 2024, Thibault Dayris"
+__email__ = "thibault.dayris@gustaveroussy.fr"
+__license__ = "MIT"
+
+from snakemake.shell import shell
+
+log = snakemake.log_fmt_shell(
```

Ruff (0.8.0)

Undefined name `snakemake`

(F821)